### DISCUSSION TITLE
Can I build only one of the defined builds?

### DISCUSSION THREAD

**Discussion by deluan** [goreleaser/goreleaser#1810] [Q&A] [answered]
Hi there,

I have a list of builds for the same binary, one for each supported platform (arm, darwin, windows,...), and would like to be able to call `goreleaser` and build the binary only for one platform. Something like this: `gorelease release --build navidrome_linux_amd64`. Is that currently possible?

By the way, here is my `.goreleaser.yml`: https://github.com/deluan/navidrome/blob/master/.goreleaser.yml

Thanks!

**Comment by caarlos0** [comment, marked as answer]
Hey, short answer: no, you'll need to edit the config file to remove the goos/goarch you don't want and then run `goreleaser build`

**Comment by deluan** [comment]
That's unfortunate.... Is this something you are planning to add in the future? If so, do you want me to create a feature request? 

Anyway, thanks for your awesome project!

**Reply by caarlos0** [reply]
> do you want me to create a feature request?

I remember there was an issue for this, but couldn't find it... feel free to open a new one, thanks :)

**Reply by tjanez** [reply]
I've opened #1852.
